src/UserWarning.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export const UserWarning: React.FC = () => (
+  <section className="section">
+    <p className="box is-size-3">
+      Please get your
```

Comment:
Instead of using multiple spaces between text and tags, consider using string interpolation or wrapping text in a single element.

Reply:
I took this file from the mate project